app/models/user.rb

```diff
@@ -4,10 +4,11 @@ class User < ApplicationRecord
   has_many :missions, dependent: :destroy
   has_secure_password
 
+  enum admin_identity: { close: 0, open: 1 }
```

Comment:
若要用 enum ，請照 Rails 慣例[設計](https://dev.to/collinjilbert/basic-enum-usage-for-defining-roles-or-statuses-on-models-in-rails-3c2f)。
```suggestion
  enum role: { user: 0, admin: 1 }
```